plugins/org.eclipse.reddeer.junit/src/org/eclipse/reddeer/junit/requirement/matcher/RequirementVersionMatcher.java

```diff
@@ -34,22 +34,17 @@
  * @author mlabuda@redhat.com
  *
  */
-public class RequirementVersionMatcher extends AbstractRequirementMatcher {
+public class RequirementVersionMatcher extends RequirementAttributeMatcher {
+	
+	private String matchingValue;
```

Comment:
Well, it will always be a string I think, bcs we read values from configuration file (whether it is json now, or yaml later...). Only other thing could be some object in case of nested configurations, but for that we have get nestedAttribute returning String value from config file. So I would keep it as it is at the moment.